provider-profile-service/src/main/java/team/proximity/provider_profile_service/about/AboutController.java

```diff
@@ -24,7 +23,7 @@ public AboutController(AboutService aboutService) {
         this.aboutService = aboutService;
     }
 
-    @Operation(summary = "Create About Company for the authenticated user")
+    @Operation(summary = "Create About Company")
     @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
     public ResponseEntity<ApiSuccessResponse> createOneAbout(@Valid @ModelAttribute AboutRequest aboutRequest) throws IOException {
```

Comment:
Handle the checked exception at the service level instead of propagating out of the controller.

context: if you can a method that throws an exception, the calling method is supposed to handle that exception.